ntbs-service/Startup.cs

```diff
@@ -175,23 +175,26 @@ public void ConfigureServices(IServiceCollection services)
             });
             SetupHangfire(services);
 
-            // DB Contexts
-            services.AddDbContext<NtbsContext>(options =>
-                options.UseSqlServer(Configuration.GetConnectionString("ntbsContext"))
-            );
+            var auditDbConnectionString = Configuration.GetConnectionString("auditContext");
 
-            services.AddSingleton<NtbsContextDesignTimeFactory>();
+            if (!Env.IsEnvironment("CI")) // Maybe we should check if connection strings are null instead?
```

Comment:
I think it's best to check on the environment (as you're doing). If we were running in any other setting and had missed out configuring the connection strings we would want to "fail fast".